### PR TITLE
Real-Time Face Mask Detector python project

### PR DESCRIPTION
**Pull request by TheKidPadra**
This is my first ever contribution to Hacktoberfest2023 and I'm so excited
I hope I did everything right 
# Problem
- In this project, I have developed a deep learning model for face mask detection using Python, Keras, and OpenCV. The face mask detector model's jobs is detecting whether a person is wearing a mask or not. I have trained the model using Keras with network architecture. Training the model is the first part of this project and testing using webcam using OpenCV is the second part.
# Solution
- This contain a convolution network that consists of two pairs of Conv and MaxPool layers to extract features from the dataset. Which is then followed by a Flatten and Dropout layer to convert the data in 1D and ensure overfitting.And then two Dense layers for classification.
- Then it's time to Image Data Generation/Augmentation
- Training the model

## Changes proposed in this Pull Request :
-  `1.`
-  `..`

## Other changes
-

### CODE REVIEW

**Comment by TheKidPadra** [comment]
What I did wrong?
Can you help me?

Screenshot 2023-10-07 at 11 08 57 AM
